### ISSUE TITLE
Unknown import when using timestamp

### ISSUE THREAD

**Issue by erizocosmico** [src-d/proteus#41]
If you use timestamp in a package, the resultant file generated by gogoproto contains this import:

```go
import _ "google/protobuf"
```

This import, obviously, doesn't exist.

**Comment by Serabe** [comment]
> Lasciati ogni speranza, voi ch'entrate

The problem is the following:

When generating the Go source from the proto file generated from our Go structs we need to add some options to the `--gofast_out".

Uhm, let me rephrase, we need to add *some undocumented options*. The thing is, we can override the imports during the compilation of Go source code from proto files. The thing works as follows:

* We can add options in the following format `M%s=%s` where the first string would be the proto import (in our case `google/protobuf/timestamp.proto`) and the second would be the go import (in our case is `github.com/gogo/protobuf/types`. These `M` options are separated by `,`.

* The set of `M` options is separated by `:` to the other out options.

Example:

```
--gofast_out="Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mserabe/wants/to/cry/because_of.proto=github.com/Serabe/really-wants-to/cry:."
```

**Comment by erizocosmico** [comment]
This was fixed, right?

**Comment by Serabe** [comment]
Effing `services`
